source/trafficshaper/limiters.rst

```diff
@@ -76,18 +76,17 @@ need to set this up manually. Take a look at Captive Portal page to set this up.
 Using Limiters for Bandwidth Guarantees
 ---------------------------------------
 
-To guarantee a certain amount of bandwidth using Limiters instead of enforcing
-an upper limit, make four limiters as follows:
+If instead of capping bandwidth usage, you wish to guarantee a certain amount of minimum bandwidth be available, use limiters to cap the "Remaining" bandwidth usage below your Total available bandwidth:
```

Comment:
Reword to remove "you" and "your" -- See https://www.netgate.com/docs/pfsense/references/style-guide.html#what-to-avoid

Reply:
thanks. done